server/routes/creatingLicences/types/address.ts

```diff
@@ -24,7 +24,8 @@ class Address extends Stringable {
   addressPostcode: string
 
   stringify(): string {
-    return Object.values(this).join(', ')
+    if (Object.values(this).join('').length > 0) return Object.values(this).join(', ')
```

Comment:
Added this so that address is not persisted to the database at all if its empty. Previously it was persisted as ", , , , "